packages/schematics/angular/ng-new/schema.d.ts

```diff
@@ -72,4 +72,8 @@ export interface Schema {
    * Skip creating spec files.
    */
   skipTests?: boolean;
+  /**
+   * Create an empty workspace with no projects.
+   */
+  empty?: boolean;
```

Comment:
just FYI we're working to make this file generated from the `.json` version

Reply:
I think that's a good idea, it was present (although manual) before, but without this the TS compilation will fail.